migrations/1665502977143-add-display-name-to-user.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { runMigration } = require('../utils/mongodb')
+
+module.exports.up = runMigration(async (db) => {
+  await db.collection('users').updateMany(
+    { displayName: { $exists: false } },
+    {
+      $set: {
+        displayName: '',
+      },
+    }
+  )
+})
+
+module.exports.down = runMigration(async (db) => {
+  await db.collection('users').updateMany({}, { $unset: { displayName: '' } })
```

Comment:
issue: this down migration will remove displayName even if it is not empty string

Suggestion: update the `updateMany` call to only apply if `displayName` is set to empty string.
Suggestion: add a couple of tests  that checks display name is not over written if set and not removed if not empty string

I point this out since I don't want to delete someone's customized name if they have selected it already if we ever need to run the down version of this migration. It could be that this is unnecessary but feels like good habit to be in.

Reply:
Great catch! I think I could just apply a filter like in the `up` function to remedy this. I'll add those tests in as well.